Write draft of communication of results section for R/E guide (#1498)
ihmeuw/vivarium_research: docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

--------------------------------------

Here we have collected recommendations from various experts and guides advocating for new standards of communication regarding racial health disparities.

1. Use a health equity lens when framing information about health disparities (https://www.cdc.gov/healthcommunication/HealthEquityGuidingPrinciples.pdf).

    Rather than allowing dominant narratives around race, individualism, and meritocracy limit our understanding of the root causes of health inequities, 
    it is important to center a health equity perspective in health metrics research.  An equity lens allows us to recognize that systems of power and oppression 
    (including white supremacy) shape institutional policies and living conditions that systematically harm populations (https://www.ama-assn.org/system/files/ama-aamc-equity-guide.pdf).

2. Use person-first language, avoid unintentional blaming, and be as specific as possible about the group to which you're referring. 
    
    For more elaboration on these tips, please see Advancing Health Equity: A Guide to Language, Narrative and Concepts (https://www.ama-assn.org/system/files/ama-aamc-equity-guide.pdf).

> 3. Explicitly define race during experimental design, and specify the reason for its use in the study (https://www.healthaffairs.org/content/forefront/racism-new-standard-publishing-racial-health-inequities). 

Reviewer: This is the part that I think most overlaps with my section! Maybe we can cite back here?

>     In defining race as it pertains to your research, you shouldreview all relevant social, environmental, and structural factors for which race may serve as a 

Reviewer: ```suggestion
    In defining race as it pertains to your research, you should review all relevant social, environmental, and structural factors for which race may serve as a 
```

>     In defining race as it pertains to your research, you shouldreview all relevant social, environmental, and structural factors for which race may serve as a 

Reviewer 2: ```suggestion
    In defining race as it pertains to your research, you should review all relevant social, environmental, and structural factors for which race may serve as a 
```

    proxy measure.  "For the reader, these additional details enable careful interpretation of study results and implications. But for authors, it engenders critical thinking about racial constructs that prevent the reification of race as a biological entity. (https://www.healthaffairs.org/content/forefront/racism-new-standard-publishing-racial-health-inequities)" 

>     Health metrics researchers should embrace a critical race theory framework, which provides "tools for conducting research and practice are intended to elucidate contemporary racial phenomena, expand the vocabulary with which to discuss complex racial concepts, and challenge racial hierarchies" (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2837428/pdf/S30.pdf)

Reviewer 2: Can we be more specific here about what this will entail?

3.0 Results
+++++++++++

3.0 Results
+++++++++++

Communication of results
------------------------

In this section we’ll discuss how health metric researchers can talk about health research related to 
race and ethnicity without stigmatizing, othering, or otherwise causing harm to historically marginalized 
racial and ethnic groups. It is important to be aware of how we talk about race and racism, 

> particularly in the field of health metrics sciences, our goal is often to uncover and scrutinize the 

Reviewer 3: ```suggestion
particularly in the field of health metrics sciences, as our goal is often to uncover and scrutinize the 
```

This sounds like a run-on sentence, but I think you can fix it by adding 'as', unless that changes your intended meaning.

underlying factors contributing to health disparities.

One helpful framework for differentiating ways of communicating about race is that of *race-based research vs. race-conscious research* 

> [Cerdena_2020]_. Race-based research, which is characterized 

Reviewer 3: ```suggestion
[Cerdeña_2020]_. Race-based research, which is characterized 
```

Reviewer 4: nit: I think the "which" and comma here should also be removed

> by race essentialism, the notion that race is a biological factor (rather than a sociopolitical construct). 

Reviewer 4: ```suggestion
by race essentialism: the notion that race is a biological factor (rather than a sociopolitical construct). 
```

The practice of using race in this way has historically been the norm in US health research, and causes harm for 

> historically marginalized and minoritized groups, thereby exacerbating the very health inequities their health 

Reviewer 4: ```suggestion
historically marginalized and minoritized groups, thereby exacerbating the very health inequities the health 
```

research may be attempting to elucidate and alleviate. For examples of outdated race-based practices in medicine, 

> and their race-conscious alternatives, see Table 1 from Cerdena et al.’s 2020 paper [Cerdena_2020]_. 

Reviewer 3: ```suggestion
and their race-conscious alternatives, see Table 1 from Cerdena et al.’s 2020 paper [Cerdeña_2020]_. 
```

Reviewer 4: ```suggestion
and their race-conscious alternatives, see Table 1 from Cerdeña et al.’s 2020 paper [Cerdeña_2020]_. 
```

> Race-conscious research, on the other hand, calls for a shift in focus from race to racism as a key determinant of 

Reviewer 4: I'd suggest swapping the order of this sentence and the previous.

illness and health.

Key recommendations from other sources
--------------------------------------

Here we have collected recommendations from various experts and guides advocating for new standards of communication 
regarding racial health disparities.

1. **Use a health equity lens when framing information about health disparities** [Calanan_2023]_.

>     Rather than allowing dominant narratives around race, individualism, and meritocracy limit our understanding of 

Reviewer 3: ```suggestion
    Rather than allowing dominant narratives around race, individualism, and meritocracy to limit our understanding of 
```

    the root causes of health inequities, it is important to center a health equity perspective in health metrics 
    research.  An equity lens allows us to recognize that systems of power and oppression (including white supremacy) 
    shape institutional policies and living conditions that systematically harm populations 
    [AMA_2021]_.

2. **Use person-first language, avoid unintentional blaming, and be as specific as possible about the group to which you're referring**. 
    
    For more elaboration on these tips, please see Advancing Health Equity: A Guide to Language, Narrative and Concepts 

>     [AMA_2021]_.

Reviewer 3: It might be helpful to give a a quick example of each of these (person-first language, avoiding unintentional blaming, and being as specific as possible), e.g. just pulling some examples directly out of the reference, so the reader doesn't have to look it up if they have no idea what it means.

When I first read it, I thought it said "first-person language," which seemed confusing as I hadn't heard of that in this context, so I searched for "first-person" in the reference and couldn't find it, of course, which was even more confusing. It took me several minutes of searching before I finally figured out it was "person-first"...

> 3. **Explicitly define race during experimental design, and specify the reason for its use in the study** [Boyd_2020]_. 

Reviewer 3: In health metrics research, we're usually not the ones designing the experiments. Would it make sense to reframe this in terms of either how we should interpret the results of studies designed by others, or how we should approach conducting our own research and analysis, given that we're not usually designing experiments _per se_?

Reviewer 3: Or if we want to cite the recommendation as is, somehow relate it back to our context of using other people's data to build simulations or regression models?

Reviewer 4: Agree, I suggest:

```suggestion
3. **Explicitly define race during \[study\] design, and specify the reason for its use in the study** [Boyd_2020]_. 
```

    
    As is described in our earlier section, `1.1 Research question`_, you should review all relevant 
    social, environmental, and structural factors for which race may serve as a proxy measure when defining 

>     race as it pertains to your research,. "For the reader, these additional details enable careful interpretation 

Reviewer 3: ```suggestion
    race as it pertains to your research. "For the reader, these additional details enable careful interpretation 
```

    of study results and implications. But for authors, it engenders critical thinking about racial constructs that 
    prevent the reification of race as a biological entity." [Boyd_2020]_ 

>     Health metrics researchers should embrace a critical race theory (CRT) framework, which provides "tools for conducting research and practice 

Reviewer 3: The word "should" here makes me wonder who is making this recommendation and why CRT is **the** thing that we should embrace. Is this your opinion that we should embrace CRT, or did you find that recommendation in one of your sources? Would it work to say "can embrace" instead, or do you want to be more forceful than that?

>     are intended to elucidate contemporary racial phenomena, expand the vocabulary with which to discuss complex racial concepts, and 

Reviewer 3: ```suggestion
    [that] are intended to elucidate contemporary racial phenomena, expand the vocabulary with which to discuss complex racial concepts, and 
```
Changing the grammar of the quotation to fit into your sentence.

    challenge racial hierarchies" [Ford_Airhihenbuwa_2010]_. Embracing a CRT framework might entail

>     some of the following: 

Reviewer: I feel like this might be missing a sentence. I kept expecting you to define CRT here. I think maybe a sentence that said CRT's core belief is XYZ and based on this, it provides LIST. Embracing this in practice might entail BULLETS.

Reviewer 3: There doesn't seem to be a succinct definition in the article, but the article lists "4 of its basic features: race consciousness, contemporary orientation, centering in the margins rather than in the mainstream, and praxis (i.e., theory-informed action)."

The Origins section says:
> Critical Race Theory is not like behavior change or epidemiological theories. Rather, it is an iterative methodology for helping investigators remain attentive to equity while carrying out research, scholarship, and practice. ... Knowledge production is the primary medium through which Critical Race Theory operates. The scholarship distinguishes contemporary racial mechanisms from older ones (e.g., Jim Crowism), expands the vocabulary for discussing racial phenomena and investigating racism effects, and explicitly incorporates the knowledge of racial and ethnic minority communities regarding marginality.

Not sure if any of that gets at what @SylLutze is looking for.

Reviewer 3: @pletale You were wondering in coworking whether the CRT section was too long. It does seem a bit long compared to the other recommendations. Here are some thoughts:

I think it should probably be its own numbered recommendation, rather than being lumped in with the "Explicitly define race during experimental design" recommendation. And I think it should probably be the _last_ numbered recommendation, or perhaps a separate paragraph after the list of recommendations, as it's describing a more comprehensive methodology while the others are more discrete things you could cross off a checklist as you're doing your research.

If you wanted to trim it down, you could probably provide a bit less detail, and do it more like the other recommendations where you list some of the basic elements, but refer the reader to the citation if they want to know more. But I also think it's fine to provide more detail if you want to leave it in, since CRT does seem to involve a lot of elements, and if we want to apply it, we need to understand what they are.

    - *Reflective practice*:  Engage in ongoing reflection about your own positions, biases, and the power dynamics in your work. This includes being open to critique, actively seeking diverse perspectives, and being committed to anti-racist practices both personally and professionally.
  

4. **Cite the experts, particularly scholars of color whose work forms the basis of the field’s knowledge on racism and its effects** 
[Boyd_2020]_.

Reporting results accurately
----------------------------

.. todo:: 

    Use recommendations from sex/gender guide to fill out this section. This section should include 
    a. Outline goals when we present our race/ethnicity-related findings
    b. General strategy (ie explain fully in standalone parts of paper, otherwise use shorthand) - Recs for detailed description of methods with examples and recs for abbreviated description with examples 

4.0 References
++++++++++++++

.. [AMA_2021]
    American Medical Assiociation. Advancing Health Equity: A Guide to Language, Narrative and Concepts. 2021. https://www.ama-assn.org/system/files/ama-aamc-equity-guide.pdf

.. [Boyd_2020]

>     Boyd RW, Lindo EG, Weeks LD, McLemore MR. On Racism: A New Standard for Publishing on Racial Health Inequities. Health Affairs Blog. 2020 Jul. doi: 10.1377/hblog20200630.939347

Reviewer 3: ```suggestion
    Boyd RW, Lindo EG, Weeks LD, McLemore MR. On Racism: A New Standard for Publishing on Racial Health Inequities. Health Affairs Blog. 2020 Jul. doi: https://doi.org/10.1377/forefront.20200630.939347
```
Despite that DOI being listed in the citation, it doesn't work when you type it into https://doi.org, but the one they have at the top of the article does.

.. [Calanan_2023]
    Calanan RM, Bonds ME, Bedrosian SR, Laird SK, Satter D, Penman-Aguilar A. CDC’s Guiding Principles to Promote an Equity-Centered Approach to Public Health Communication. Prev Chronic Dis 2023;20:230061. DOI: http://dx.doi.org/10.5888/pcd20.230061

> .. [Cerdena_2020]

Reviewer 3: ```suggestion
.. [Cerdeña_2020]
```
I'm going to try to ñ-ize the citations too... hopefully I don't screw anything up. If you want to be safe, you could make the edit locally then push it, instead of merging my suggestions on GitHub.

>     Cerdena JP, Plaisime MV, Tsai J. The Lancet. 2020 Oct; 396(10257):1125-1128. doi: https://doi.org/10.1016/S0140-6736(20)32076-6

Reviewer 3: ```suggestion
    Cerdeña JP, Plaisime MV, Tsai J. The Lancet. 2020 Oct; 396(10257):1125-1128. doi: https://doi.org/10.1016/S0140-6736(20)32076-6
```
Might as well use an actual eñe.

.. [Ford_Airhihenbuwa_2010]

>     Ford CL, Airhihenbuwa CO. Critical Race Theory, Race Equity, and Public Health: Toward Antiracism Praxis

Reviewer 3: Maybe add a link to the article here, and/or the DOI:

PubMed Central: https://pmc.ncbi.nlm.nih.gov/articles/PMC2837428/

DOI: https://doi.org/10.2105/AJPH.2009.171058

Reviewer 3: Oh, and also a more complete citation:

> Ford CL, Airhihenbuwa CO. Critical Race Theory, race equity, and public health: toward antiracism praxis. Am J Public Health. 2010 Apr 1;100 Suppl 1(Suppl 1):S30-5. doi: 10.2105/AJPH.2009.171058. Epub 2010 Feb 10. PMID: 20147679; PMCID: PMC2837428.

And then you can make the DOI and the PMID into hyperlinks using the links above.
